android/Reviewer/app/src/main/java/rs/reviewer/rest/BaseService.java

```diff
@@ -9,7 +9,10 @@
 public class BaseService {
 
     //EXAMPLE: http://192.168.43.73:8080/rs.ftn.reviewer.rest/rest/proizvodi/
-    public static final String SERVICE_IP_ADDRESS = "192.168.1.103";
+    //kristina 192.168.1.103
```

Comment:
da ne dodje do zabune, nije to moja IP adresa - ja svoju nisam pushovala (nevenina je)